weave/flow/scorer.py

```diff
@@ -46,20 +46,24 @@ def auto_summarize(data: list) -> dict[str, Any] | None:
     """
     if not data:
         return {}
-    data = [x for x in data if x is not None]
+
     val = data[0]
 
     if isinstance(val, (bool, box.BoxedBool)):
         return {
             "true_count": (true_count := sum(1 for x in data if x)),
-            "true_fraction": true_count / len(data),
+            "true_proportion": true_count / len(data),
```

Comment:
You can't go changing the term here, that will break comparison to all the old evals people have done.

_fraction is also a better choice IMO, I've used that for a long time for this type of thing and people understand it. Originally recommended to me by a principle engineer at Google as the "right term" for this :)